src/main/java/uk/ac/ebi/embl/api/translation/CdsTranslator.java

```diff
@@ -140,17 +140,17 @@ public ExtendedResult<TranslationResult> translate(CdsFeature cds, Entry entry)
 
       if (translationResult.isFixedLeftPartial()) {
         if (!cds.getLocations().isComplement()) {
-          cds.getLocations().setLeftPartial(translator.isLeftPartial());
+          cds.getLocations().setFivePrime(translator.isLeftPartial());
```

Comment:
Potentially unsafe. Best would be to review change Translation left and right partiality as well.